apps/ca.c

```diff
@@ -1297,7 +1297,7 @@ int ca_main(int argc, char **argv)
             if (!rotate_index(dbfile, "new", "old"))
                 goto end;
 
-            BIO_printf(bio_err, "Data Base Updated\n");
+            BIO_printf(bio_err, "Database Updated\n");
```

Comment:
I think if we're going to update "Data Base" to "Database", we should also update "Updated" to "updated".

These aren't titles, they are just notices to say what the program has done, so normal sentence case is appropriate.

Reply:
Done.